src/include/duckdb/core_functions/aggregate/algebraic/covar.hpp

```diff
@@ -53,15 +53,17 @@ struct CovarOperation {
 		if (target.count == 0) {
 			target = source;
 		} else if (source.count > 0) {
-			const auto count = target.count + source.count;
-			const auto meanx = (source.count * source.meanx + target.count * target.meanx) / count;
-			const auto meany = (source.count * source.meany + target.count * target.meany) / count;
+			const auto target_count = static_cast<double>(target.count);
+			const auto source_count = static_cast<double>(source.count);
+			const auto count = static_cast<double>(target_count + source.count);
+			const auto meanx = (source_count * source.meanx + target_count * target.meanx) / count;
+			const auto meany = (source_count * source.meany + target_count * target.meany) / count;
 
 			//  Schubert and Gertz SSDBM 2018, equation 21
 			const auto deltax = target.meanx - source.meanx;
 			const auto deltay = target.meany - source.meany;
-			target.co_moment =
-			    source.co_moment + target.co_moment + deltax * deltay * source.count * target.count / count;
+			target.co_moment = source.co_moment + target.co_moment +
+			                   deltax * deltay * static_cast<double>(source.count * target.count) / count;
```

Comment:
We have our own formatter (`make format-head`) - did you editor do this?

Reply:
Found the problem here, our formatter for some reason is not run on that file, I will add them back.